test/forms/labels/input-label.test.js

```diff
@@ -1,93 +1,105 @@
 import React from 'react'
 import { mount } from 'enzyme'
 import { InputLabel } from '../../../src/'
+import { render, screen } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
 
 const name = 'person.firstName'
 const formattedName = 'First Name'
 
 test('when label is false - does not render a label', () => {
-  const wrapper = mount(<InputLabel name={name} label={false} />)
-  expect(wrapper.find('label').exists()).toEqual(false)
+  render(<InputLabel name={name} label={false} />)
+  const label = screen.queryByText(formattedName)
+  expect(label).toBeNull()
 })
 
 test('when label not provided - renders a label with content equal to formatted input name', () => {
-  const wrapper = mount(<InputLabel name={name} />)
-  expect(wrapper.find('label').text()).toEqual(formattedName)
+  render(<InputLabel name={name} />)
+  const label = screen.getByText(formattedName)
+  expect(label).not.toBeEmptyDOMElement()
```

Comment:
What's the difference between `toBeIntheDocument()` and `not.toBeEmptyDOMElement()`? I think by nature of the fact that it's able to find the element by text that we know it's not empty, right?